### DISCUSSION TITLE
Issue History: Changing status of an issue added to a Projects Beta project does not log to the issue history

### DISCUSSION THREAD

**Discussion by jm-pi** [community/community#35833] [Projects and Issues]
When an issue is added to a Classic Project, any time the status is changed, that change is logged to the issue history (green arrows):
![image](https://user-images.githubusercontent.com/70594789/195212068-251a113d-16c4-463d-9473-c8a5f2e52d03.png)

When an issue is added to a new Projects Beta project, status changes are not logged. This is a different issue, and the red arrows here reflect where the issue should have been logged as being added to the new beta project To Do status (first arrow) and when the dev moved it from To Do to In Development (second arrow):
![image](https://user-images.githubusercontent.com/70594789/195212568-94d17804-f974-44d9-84a0-d959dc0a7e9a.png)

If it helps, both of the issues in these screenshots are from the same repo. We are in the process of shifting our issue tracking to the new Projects Beta, as it offers much more functionality for our workflows, so everything is otherwise the same as the classic project that we're shifting issues that are still open out of before closing the classic project and moving forward with the new version of the same project.

I'm not sure if there's just some setting somewhere I'm not seeing, or if this is a bug that has not yet been addressed, but we like being able to see exactly when our devs begin working on an issue assigned to them, so it's very helpful to be able to see those status changes in the issue history, and any assistance that could be provided on this would be fantastic.

Thank you!!

**Comment by mmcaulay** [comment]
I see it as a feature they haven't completed rather than a bug, definitely wish it did that too!
